HEC.FDA.Model/structures/Structure.cs

```diff
@@ -107,7 +99,7 @@ public ConsequenceResult ComputeDamage(float waterSurfaceElevation, List<Determi
             if (analysisYear > YearInService)
             {
                 //Beginning damage depth is relative to the first floor elevation and so a beginning damage depth of -1 means that damage begins 1 foot below the first floor elevation
-
+                //if not defined by the user, the beginning damage depth is equal to the negative of foundation height
```

Comment:
is this standard? i guess it is a decision, but typically i assume beginning damage is foundation height not ground.

Reply:
it is common in depth-percent damage functions for percent damage to be positive below the first floor - I think it we assumed beginning damage was foundation height/FFE, we'd truncate that damage, right?